docs/apache-airflow/howto/dynamic-dag-generation.rst

```diff
@@ -29,7 +29,7 @@ If you want to use variables to configure your code, you should always use
 `environment variables <https://wiki.archlinux.org/title/environment_variables>`_ in your
 top-level code rather than :doc:`Airflow Variables </concepts/variables>`. Using Airflow Variables
 at top-level code creates a connection to metadata DB of Airflow to fetch the value, which can slow
-down parsing and place extra load on the DB. See the `Airflow Variables <_best_practices/airflow_variables>`_
+down parsing and place extra load on the DB. See the `Airflow Variables <https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#airflow-variables>`_
```

Comment:
I think something like that is beter (not 100% sure - you need to look it up as it is done in other places). The .rst files are processed by sphinx so if we are referring to other document in airflow docs we should use `:doc:` directive then when documentation is built, the right link will be generated. This is important because for example it will use the link to the right version of Airflow doc. 

```suggestion
down parsing and place extra load on the DB. See the :doc:`Airflow Variables <_best_practices/airflow_variables>`
```